ide-support/lsp/lsp-server/src/main/java/io/helidon/lsp/server/service/ContentManager.java

```diff
@@ -121,7 +124,9 @@ private String createNewTempFile(String fileName) {
             Files.write(tmp, content);
             return tmp.toUri().toString();
         } catch (URISyntaxException | IOException e) {
-            LOGGER.log(Level.SEVERE, "Cannot create temp file for {0}. Exception - {1}", new Object[]{fileName, e.getMessage()});
+            String message = String.format("Cannot create temp file for %s. Exception - %s", fileName, e.getMessage());
```

Comment:
Shouldn't these be wrapped with runtime exceptions ?

Reply:
Why do we need to throw a runtime exception? The problem will be logged and the user will see the message about it in VSCode.